packages/ebayui-core-react/.github/actions/deploy/action.yml

```diff
@@ -12,12 +12,12 @@ runs:
     steps:
         - name: Build Storybook
           shell: sh
-          run: NODE_OPTIONS=--openssl-legacy-provider yarn build-storybook --output-dir storybook-static/${{ inputs.branch }}
+          run: NODE_OPTIONS=--openssl-legacy-provider npm run build:storybook -- --output-dir ./_site/${{ inputs.branch }}
 
         - name: Publish Storybook
           uses: JamesIves/github-pages-deploy-action@v4
           with:
-              folder: storybook-static
+              folder: _site
```

Comment:
We should remove this action
For deploy it should only run the root one
Our deploy package.json action should call the deploy command for each package and then it copies all the directories to the root `_site`